### PR TITLE
Add missing share_via_native_mail feature

### PR DESCRIPTION
**Pull request by human-spider**

### CODE REVIEW

File: sdk/src/main/java/com/talkable/sdk/Talkable.java [hunk 1]
```diff
@@ -181,12 +181,16 @@ private static void setNativeFeatures(Context context) {
             isMessengerInstalled = MessengerUtils.hasMessengerInstalled(context);
         }
 
+        Intent sendNativeMailIntent = new Intent(Intent.ACTION_SENDTO, Uri.parse("mailto:"));
+        Boolean canSendNativeMail = !context.getPackageManager().queryIntentActivities(sendNativeMailIntent, 0).isEmpty();
```
Comment (dgtv) [inline sdk/src/main/java/com/talkable/sdk/Talkable.java]: Please move it under `if (context != null)` and set default value to `false` as `isSmsAvailable & isMessengerInstalled ` do

Comment (human-spider) [reply sdk/src/main/java/com/talkable/sdk/Talkable.java]: `null` is a bane of Java

**Comment by vitalikdanchenko** [comment]
@dgtv could you please take care of this pull request?

**Comment by dgtv** [comment]
Closed in favor of https://github.com/talkable/android-sdk/pull/6

**Pull request closed without merging**